### PR TITLE
create release `3.40.1`

### PR DESCRIPTION
**Pull request by dixler**
# Description

I will have to make a follow up PR to bump from 3.40.1 -> 3.40.2 in order to block the merge queue from changing 3.40.1

Fixes # (issue)

## Checklist

- [ ] I have added tests that prove my fix is effective or that my feature works

- [ ] I have run `make changelog` and committed the `changelog/pending/<file>` documenting my change

- [ ] Yes, there are changes in this PR that warrants bumping the Pulumi Service API version

### CODE REVIEW

File: .version [hunk 1]
```diff
@@ -1 +1 @@
-3.41.0
+3.41.1
```
Comment (dixler) [inline .version:1]:
```suggestion
3.40.1
```

**Review by t0yv0 — APPROVED** [review]

**Comment by dixler** [comment]
bors merge

**Comment by t0yv0** [comment]
ACtually taking approval back - a bit distracted today and I'm not fully sure how the new release system works. Let @AaronFriel review for now.

**Comment by t0yv0** [comment]
Is the version going down OK?

**Comment by AaronFriel** [comment]
Yeah

**Review by AaronFriel — APPROVED** [review]

**Pull request merged**